esx_service/vmci/vmci_client.c

```diff
@@ -161,9 +165,10 @@ dummy_get_reply(be_sock_id *id, be_request *r, be_answer* a)
 static be_sock_status
 vsock_init(be_sock_id *id, int cid, int port)
```

Comment:
well, the code does change `id->addr.svm_port = port` to `id->addr.svm_port = round_robin`, so now I  not sure how it could have been the ESX  service port